prod/westeurope/common/cdn/cdn_endpoint_resources/terragrunt.hcl

```diff
@@ -0,0 +1,28 @@
+dependency "cdn_profile" {
+  config_path = "../cdn_profile"
+}
+
+dependency "storage_account_static_assets_iopay" {
+  config_path = "../storage_account_resources"
+}
+
+# Common
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_cdn_endpoint?ref=v2.1.15"
+}
+
+inputs = {
+  name                = "resources"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+  profile_name        = dependency.cdn_profile.outputs.resource_name
+  origin_host_name    = dependency.storage_account_resources.outputs.primary_web_host
```

Comment:
Here the dependency is: dependency.storage_account_static_assets_iopay
I'll fix it

Reply:
@uolter this is not for the project io-pay, but to host the static files of the web site io.italia.it. Please see the attached story https://www.pivotaltracker.com/story/show/176008623

Reply:
@uolter Morover, I've seen that the `cdn` already has an endpoint named `asset`. Wouldn't it be better to reuse it?

Reply:
@AleDore can you please help @GiovanniMancini ? Could he use the same storage account and cdn enpoint?

Reply:
@AleDore I think the endpoint `assets` on the cdn is no longer used, so I think we can reuse it with a new storage account. Do you agree?